package.json

```
@@ -15,8 +15,8 @@
   "author": "",
   "license": "ISC",
   "engines": {
-    "node": ">=14 <20",
-    "npm": ">=7.20.x <9"
+    "node": ">=18 <20",
```

we should probably still support node 16, as it has both been tested to work with this repo and hasn't reached end of life yet. we might want to consider expanding npm to include node 20 as that is now current.

```suggestion
    "node": ">=16 <=20",
```